packages/spark-core/src/plugins/credentials/auth-interceptor.js

```diff
@@ -54,12 +54,12 @@ export default class AuthInterceptor extends Interceptor {
   }
 
   /**
-   * @see Interceptor#onRequestError
+   * @see Interceptor#onResponseError
    * @param {Object} options
    * @param {Error} reason
    * @returns {Object}
    */
-  onRequestError(options, reason) {
+  onResponseError(options, reason) {
```

Comment:
it's called automatically. the problem was that i've had it named incorrectly since December and therefore it never got called.